sympy/polys/matrices/domainmatrix.py

```diff
@@ -170,18 +258,33 @@ def unify(self, other):
         return self, other
 
     def to_Matrix(self):
-        """ Convert DomainMatrix to Matrix
+        r"""
+        Convert DomainMatrix to Matrix
+
+        Returns
+        =======
+
+        Matrix
+            MutableDenseMatrix for the DomainMatrix
 
         Examples
         ========
 
         >>> from sympy import ZZ
         >>> from sympy.polys.matrices import DomainMatrix
-        >>> A = DomainMatrix([[ZZ(1), ZZ(2)], [ZZ(2), ZZ(3)]], (2, 2), ZZ)
-        >>> A.to_Matrix()
+        >>> domainmatrix = DomainMatrix([
+        ...    [ZZ(1), ZZ(2)],
+        ...    [ZZ(3), ZZ(4)]], (2, 2), ZZ)
+
+        >>> domainmatrix.to_Matrix()
         Matrix([
-        [1, 2],
-        [2, 3]])
+            [1, 2],
+            [3, 4]])
```

Comment:
I think variable name A is simpler so just let this example also remain as it is.

Reply:
Will do.